src/main/java/com/troubleshooting/model/QueryModel.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Model object for user input
+ */
+package com.troubleshooting.model;
+
+public class QueryModel implements IQueryModel{
+    private String containerId;
+    private String query;
+
+    @Override
+    public String getContainerId() {
+        return containerId;
+    }
+
+    @Override
+    public void setContainerId(String containerId) {
+        this.containerId=containerId;
+    }
+
+    @Override
+    public String getQuery() {
+        return query;
+    }
+
+    @Override
+    public void setQuery(String query) {
+        this.query=query;
```

Comment:
spacing

Reply:
should this also have a constructor instead of getters/setters?